flycheck.el

```diff
@@ -4310,6 +4360,11 @@ non-nil."
     (cancel-timer flycheck-display-error-at-point-timer)
     (setq flycheck-display-error-at-point-timer nil)))
 
+(defun flycheck-automatically-display-error-at-point ()
```

Comment:
I don't think we really need this function, since if the `defcustom` is nil, the hooks will never be called, and the only other place we call display-error, we can guard it directly (see above).